src/ElseRenderContent.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ElseRenderContent: React.FC = ({ children }: React.PropsWithChildren<{}>) => (
```

Comment:
My question here is if this is necessary to have another file to render the content

Reply:
It's not, I've just removed it.